src/program.cpp

```diff
@@ -15,7 +15,6 @@
 #include <algorithm>
 #include <cstring>
 #include <filesystem>
-#include <iostream>
```

Comment:
why did you remove that include?

Reply:
because I think i put this in accidentally in a past cl but lets leave it there.